src/ports/lists/component.ts

```diff
@@ -113,16 +122,31 @@ export function createListsComponent(
     }
   }
 
-  async function getLists(params: GetAuthenticatedAndPaginatedParameters): Promise<DBGetListsWithCount[]> {
-    const { userAddress, limit, offset } = params
-    // TODO: do we want to sort the lists using another criteria?
-    const result = await pg.query<DBGetListsWithCount>(SQL`
+  async function getLists(params: GetListsParameters): Promise<DBGetListsWithCount[]> {
+    const { userAddress, limit, offset, sortBy = ListSortBy.NEWEST, sortDirection = ListSortDirection.DESC } = params
+    const query = SQL`
         SELECT l.*, COUNT(*) OVER() as lists_count, user_address = ${DEFAULT_LIST_USER_ADDRESS} as is_default_list
         FROM favorites.lists l
         WHERE user_address = ${userAddress} OR user_address = ${DEFAULT_LIST_USER_ADDRESS}
-        ORDER BY is_default_list DESC
-        LIMIT ${limit} OFFSET ${offset}
-    `)
+        `
+    const orderByQuery = SQL`ORDER BY is_default_list DESC`
+
+    switch (sortBy) {
+      case ListSortBy.NEWEST:
+        orderByQuery.append(SQL`, created_at DESC`)
+        break
+      case ListSortBy.NAME:
+        orderByQuery.append(SQL`, name ${sortDirection}`)
+        break
+      case ListSortBy.OLDEST:
+        orderByQuery.append(SQL`, created_at ASC`)
+        break
+    }
```

Comment:
Although in the UI, the newest and oldest sorts are common, when requesting the backend services, we could parametrize the query as we want. We could use the `sortDirection` alongside the sortBy `date` to simplify how the backend works.

```suggestion
    switch (sortBy) {
      case ListSortBy.DATE:
        orderByQuery.append(SQL`, created_at ${sortDirection}`)
        break
      case ListSortBy.NAME:
        orderByQuery.append(SQL`, name ${sortDirection}`)
        break
    }
```
What do you think about it?